apps/cms/settings/models.py

```diff
@@ -91,15 +91,24 @@ class ImportantPages(BaseSetting):
 
 
 # these are settings for platform organisation
-@register_setting
+@register_setting(icon='pick')
 class OrganisationSettings(BaseSetting):
     platform_name = models.CharField(
         max_length=20,
         default="adhocracy+",
         verbose_name="Platform name",
+        help_text=("This name appears in the footer of all pages and e-mails "
+                   "as well as in the tab of the browser.")
+    )
+    address = fields.RichTextField(
+        help_text="The address is published on the contact form."
     )
-    address = fields.RichTextField()
-    contacts = fields.RichTextField()
+    contacts = fields.RichTextField(
+        help_text="The contacts are published on the contact form."
+    )
+
+    class Meta:
+        verbose_name = 'Platform settings'
```

Comment:
Should we make it translatable (also for important pages)? Because the rest of the settings options show in German, I suppose coming from wagtail?

Reply:
Yeah, I also thought about that, but all the wagtail we added, also in the page models wasn't translated, so I didn't do that.

Reply:
Ok. Just thought it looks weird in the menu with a mix of German and English, but as its a mix everywhere, probably it doesnt matter.